cmd/kubeadm/app/node/bootstrap.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang/glog"
```

Comment:
I'm afraid glog produces very noisy output and we aren't using it in other places, could please replace it with just a simple `fmt.Println`? We can can figure out how to change log formatter later and switch to glog then...